modules/deploying-lvms-on-sno-cluster.adoc

```diff
@@ -22,6 +22,11 @@ You can deploy {lvms} using one of the following:
 * {rh-rhacm-first}
 * {product-title} Web Console
 
+[WARNING]
+====
+{lvms} only supports provisioning local storage on multi-node clusters. It does not support storage data replication mechanism across nodes. When you are using {lvms} on multi-node clusters, you must ensure storage data replication through active or passive replication mechanism to avoid single point of failure.
```

Comment:
```suggestion
{lvms} only supports provisioning local storage on multi-node clusters. It does not support storage data replication mechanism across nodes. When you are using {lvms} on multi-node clusters, you must ensure storage data replication through active or passive replication mechanism to avoid a single point of failure.
```

Reply:
Updated as per the suggestion.